server/.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@1.8.0
+
+jobs:
+  buildAndUnitTest:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle build
+  acceptanceTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run:
+          command: gradle run
+          background: true
+      - run: git submodule update --init --recursive
+      - ruby/install:
+          version: '2.6.8'
+      - run:
+          command: |
+            cd http_server_spec
+            gem install bundler
+            bundle install --path vendor/bundle
+            sudo apt install lsof
+            bundle exec spinach --tags @01-getting-started
```

Comment:
Question: What is this for?

Reply:
We put `circleCI` in the wrong `.gitIgnore` file. We nested it in the subtree'ed gitIgnore. Instead of the main apps. cc. @jonmanahan well get that changed and update you with the commit